### PR TITLE
Issue #46: Removed @botkube help commands. Added Makefile to add support for git tags and docker build with versioning

### PR DESCRIPTION
**Pull request by mugdha-adhav**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+#Tag images
+tag-image: 
+	@echo 'Tagging image'
+	@docker tag $(IMAGE_REPO) $(IMAGE_REPO):$(TAG)
+	@docker tag $(IMAGE_REPO) $(IMAGE_REPO):latest
+
+#Docker push image
+publish:
+	@echo "Pushing docker image to repository"
+	@docker login
```
[the remark this answers is not in the document]
Comment (mugdha-adhav) [reply Makefile]: If not already logged in, asks for user-name and password.

[the remark this answers is not in the document]
Comment (ssudake21) [reply Makefile]: How does it work with Travis ?

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+#Docker Tasks
+#Make a release
+release: git-tag build tag-image publish 
+	@echo "Successfully released version $(TAG)"
+
+#Create a git tag
+git-tag: check-git-status
+	@echo "Creating a git tag"
+	@git add .
+	@git commit -m "Bumped to version $(TAG)" ;
```
[the remark this answers is not in the document]
Comment (mugdha-adhav) [reply Makefile]: Yes we can remove the commit. Will test and update in next commit.

File: Makefile (hunk 1 above)
[the remark this answers is not in the document]
Comment (mugdha-adhav) [reply Makefile]: Made changes in Travis to add docker build.

**Pull request merged**